usr/local/sbin/lncm-createwallet.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+'''
+Copyright © 2018-2019 LNCM Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import base64, codecs, json, requests, os
+import random, string
+
+# Generate seed
+url = 'https://localhost:8181/v1/genseed'
+# Initialize wallet
+url2 = 'https://localhost:8181/v1/initwallet'
+cert_path = '/media/important/lnd/tls.cert'
+seed_filename  = '/home/lncm/seed.txt' 
+
+
+''' 
+  Functions have 2 spaces
+'''
+def randompass(stringLength=10):
```

Comment:
Maybe not for now, but we should somehow "centralise" password _generation_, so that later it can be easily replaced with password _derivation_.

Reply:
thats not very "decentralised" of you :P